modules/apigee_edge_apidocs/src/Entity/ApiDoc.php

```diff
@@ -122,16 +139,48 @@ public function setCreatedTime(int $timestamp) : ApiDocInterface {
   /**
    * {@inheritdoc}
    */
-  public function isPublished() : bool {
-    return (bool) $this->getEntityKey('status');
+  public function getRevisionUser() {
```

Comment:
Why did you override the inherited implementation from `\Drupal\Core\Entity\RevisionLogEntityTrait::getRevisionUser()`?

Reply:
My bad, I have now removed the overriden method.